.github/workflows/tests.yml

```diff
@@ -210,9 +210,22 @@ jobs:
       - name: Check dependencies
         run: |
           ./.ci.sh -c
-      - name: Run pytest test suite
-        run: |
-          ./.ci.sh -t
+      - name: Run pytest test suite (with code coverage)
+        run: |  # NB: We use `-v` instead of `-t` here to run this one with coverage
```

Comment:
Ah, thank you for pointing this out! 

Just to provide some context, the `batch_processing` tests act as a much bigger integration test, chaining multiple `sct` commands together in a pipeline shell script. This requires about ~15-30m of dedicated processing before any results can actually be validated. So, to save time, we run them in a separate CI workflow, parallel to the more granular unit tests.

It's not very clear by just looking at the folders (sorry about that), but the tests in `testing/batch_processing` will only be run if a specific environment variable is set first. And, we set that environment variable via the [`test-batch-processing.yml`](https://github.com/spinalcordtoolbox/spinalcordtoolbox/blob/master/.github/workflows/test-batch-processing.yml) CI workflow that manages the `batch_processing` tests.

So, right now `-t` will run the same tests as `-v` (even though that isn't very clear!)

Reply:
Typing this out, though, it isn't very clear that batch_processing gets skipped by `-v`. There might be a better way of  organizing things so that it is more apparent. :thinking:

Reply:
Perhaps we could update `-t` to explicitly run `pytest` on `testing/api` and `testing/cli` via 2 separate commands, too? That way, if we _do_ add tests in a new folder, we'll be forced to make updates to account for the new folder, and thus will be less likely to silently miss running a test.

Reply:
Thanks for the context. Changing `-t` to be explicit like `-v` would be clearer, yes. Although it doesn't have to be two separate commands, just `pytest testing/api testing/cli` would be fine.